ExtLibs/ArduPilot/Mavlink/MAVLinkInterface.cs

```diff
@@ -5221,7 +5221,7 @@ public async Task<MAVLinkMessage> readPacketAsync()
                         if (MAVlist[sysid, compidcurrent].cs.version.Major > 0 ||
                             MAVlist[sysid, compidcurrent].cs.version.Minor >= 4)
                         {
-                            if (sev <= (byte)MAV_SEVERITY.WARNING)
+                            if (sev <= (byte)Settings.Instance.GetInt32("severity"))

```

Comment:
can you add the default value. as if this has never been set by the user, it will results in a 0 atm

Reply:
I would add processing-ization.